### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - New Cards] - Ensures every form element has a label

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8357]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/samples/ActivityUpdate.html
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to 'Designer' tab item and press enter key to activate it.
3. Navigate to the 'New Card' button and press enter key to activate it.
4. Run accessibility insight for web tool and observe the issue.

**Actual Result:**
Under the 'Application Login' form field does not have label as screen reader is announcing only as 'Edit'.

**Expected Result:**
Under the 'Application Login' for the Username and Password field should have label so that screen reader should convey the proper information about the text edit fields.

**Element path:**
 #__ac-81a777f2-9d82-46cd-bc9d-89e142f2bd0d

**How to fix:** 
Fix any of the following:
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

**User Impact:**
It will impact the screen reader user if the edit field does not have label or placeholder text as they won't be able to know the exact information about the elements.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![MAS4 1 2-Ensure every form element has a label](https://user-images.githubusercontent.com/95913748/224275803-7c046b1b-b4a8-4550-8ee0-6a6fd8bc19e4.png)

![MAS4 1 2- Ensures every form element has a labels](https://user-images.githubusercontent.com/95913748/224275909-102d7743-f211-4b5b-bb31-dc5fd1c193bc.png)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

Closing this issue because it does not repro on the latest release of our website. Please let me know if you have any questions!

**Comment by vagpt** [comment]
This issue is not repro on the below environment, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1
